src/React/Renderers/VtkGeometryRenderer/index.js

```diff
@@ -131,45 +133,35 @@ export default class VtkGeometryRenderer extends React.Component {
     }
   }
 
-  addViewObserver(viewId) {
-    this.props.client.VtkGeometryDelivery.addViewObserver(viewId).then((successResult) => {
-      console.log(`Successfully added observer to view ${viewId}`);
-      console.log(successResult);
-      this.setState({ viewId: successResult.viewId });
-    }, (failureResult) => {
-      console.log(`Failed to add observer to view ${viewId}`);
-      console.log(failureResult);
-    });
+  getCameraParameters() {
+    // Calling getCurrentRenderer() on the interactorStyle seems to return null
+    // const activeCamera = this.interactorStyle.getCurrentRenderer().getActiveCamera();
+    const activeCamera = this.renderWindow.getRenderers()[0].getActiveCamera();
+    const cameraParams = activeCamera.get('position', 'focalPoint', 'viewUp');
+    return Object.assign({}, cameraParams, { centerOfRotation: this.interactorStyle.getCenterOfRotation() });
+  }
+
+  setCameraParameters(params) {
+    // Calling getCurrentRenderer() on the interactorStyle seems to return null
+    // const activeCamera = this.interactorStyle.getCurrentRenderer().getActiveCamera();
+    const activeCamera = this.renderWindow.getRenderers()[0].getActiveCamera();
+    activeCamera.set({ position: params.position });
+    activeCamera.set({ focalPoint: params.focalPoint });
+    activeCamera.set({ viewUp: params.viewUp });
+    this.interactorStyle.set({ centerOfRotation: params.centerOfRotation });
+    this.renderWindow.render();
```

Comment:
This could be written

```
setCameraParameters({ position, focalPoint, viewUp, centerOfRotation }) {
    const activeCamera = this.renderWindow.getRenderers()[0].getActiveCamera();
    activeCamera.set({ position, focalPoint, viewUp });
    this.interactorStyle.setCenterOfRotation(centerOfRotation);
   this.renderWindow.render();
}
```